vignettes/git_usage.Rmd

```diff
@@ -19,6 +19,10 @@ knitr::opts_chunk$set(
 
 admiral GitHub: https://github.com/Roche-GSK/admiral.git
 
+# Introduction
+
+This article will give you an overview of how the admiral project is utilizing the version-control software git and the website Github while working with RStudio.  We will go over the primary branches that house the source code for the admiral project as well as how we use **Feature** branches to address Issues.  Issues can range from bugs to enhancement that have been identified or requested by developers, users or testers.  Finally, we provide the bare minimum git commands needed to get up and running.  Please refer to the Resource section for more in-depth guidance on using git and Github.  
```

Comment:
Please write it as Git**H**ub.

Reply:
Also please always wrap git in backticks such that it rendes like `git`.

Reply:
You seem to be using two spaces after a period which is contrary to the existing text. Please replace by only a single one for consistency.